packages/scandipwa/src/component/ProductAttributeValue/ProductAttributeValue.component.js

```diff
@@ -269,20 +269,34 @@ export class ProductAttributeValue extends PureComponent {
         );
     }
 
-    getCheckboxLabel = (label, subLabel) => (
-        <span
-          block="ProductAttributeValue"
-          elem="Label"
-        >
-            { label }
+    renderSublabel = (subLabel) => {
+        if (!subLabel) {
+            return null;
+        }
+
+        return (
             <strong
               block="ProductAttributeValue"
               elem="SubLabel"
             >
-                { `(${subLabel})` }
+              { `(${subLabel})` }
             </strong>
-        </span>
-    );
+        );
+    };
+
+    getCheckboxLabel = (label) => {
+        const { isProductCountVisible } = this.props;
+
+        return (
+            <span
+              block="ProductAttributeValue"
+              elem="Label"
+            >
+                { label }
+                { isProductCountVisible && this.renderSublabel() }
```

Comment:
Instead of checking it here, you can get prop isProductCountVisible in function renderSublabel and check it there. Like:
```
 if (!isProductCountVisible || !subLabel) 
 ```
 
 Same result just bit more clear